TrueCraft/Commands/DebugCommands.cs

```diff
@@ -94,12 +94,30 @@ public override string[] Aliases
 
         public override void Handle(IRemoteClient Client, string Alias, string[] Arguments)
         {
-            if (Arguments.Length != 0)
+            switch (Arguments.Length)
             {
-                Help(Client, Alias, Arguments);
-                return;
+                case 1:
+                    Client.SendMessage(Client.World.Time.ToString());
+                    break;
+                case 3:
+                    if (!Arguments[1].Equals("set"))
+                        Help(Client, Alias, Arguments);
+
+                    int newTime;
+
+                    if(!Int32.TryParse(Arguments[2], out newTime))
+                        Help(Client, Alias, Arguments);
+
+                    Client.World.Time = newTime;
+
+                    Client.SendMessage(string.Format("Setting time to {0}", Arguments[2]));
+                    Client.QueuePacket(new TimeUpdatePacket(newTime));
```

Comment:
You'll want to send this packet to all clients in that world, not just the one issuing the command.